DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/SwiftUI/SwiftUIWireframesBuilder.swift

```diff
@@ -70,6 +74,10 @@ internal struct SwiftUIWireframesBuilder: NodeWireframesBuilder {
         case let .clip(path, _):
             let clip = context.convert(frame: path.boundingRect)
             context.clip = context.clip.intersection(clip)
+        case let .filter(filter):
+            if case .colorMultiply(let color) = filter {
+                context.tintColor = color
+            }
```

Comment:
nit/ Would that work?
```suggestion
        case let .filter(.colorMultiply(color)):
```